node/coinstacks/solana/api/src/models.ts

```diff
@@ -51,3 +51,10 @@ export interface PriorityFees {
   // average confirmation speed estimation
   fast: number
 }
+
+/**
+ * Returns the gas fees estimation
+ */
+export interface GasFeesBody {
```

Comment:
```suggestion
export interface EstimateFeesBody {
```

Reply:
Fixed in [d50be64](https://github.com/shapeshift/unchained/pull/1051/commits/d50be6415f5b5822023b295cfe71938ed521ffb5)